XenModel/XenAPI-Extensions/Host.cs

```diff
@@ -866,6 +866,15 @@ public List<XenRef<VM>> GetRunningPvVMs()
             return vms.ToList();
         }
 
+        public List<VM> GetPvVMs()

```

Comment:
I think it will make for a neater design if we add a method in `XenAPI-Extensions/VM.cs`, e.g.
```
public bool IsPvVm()
{
    return is_a_real_vm() && !IsHVM() && !other_config.ContainsKey("pvcheckpass");
}
```
and call that one from where it's needed.

The PVGuestsCheck  has been added as a host check, but the current method is not host specific since it's looking for VMs at the connection level - or do you rather mean VMs with the specific host as Home()? Also, do we care about real VMs only or custom templates and snapshots too? I think we need to clarify the specification/design a bit.